### DISCUSSION TITLE
Last Date Confirmation

### DISCUSSION THREAD

**Discussion by TiwariAbhishek23** [DiveHQ/backend-internship-task#28] [Q&A]
Hello there! I wanted to confirm if today, June 19 at 12:00 AM IST, is the deadline for submissions. I have already completed my submission, but I'm still working on improving it. Could you kindly provide me with the correct information? Thank you for your assistance.

**Comment by 0xkarambit** [comment]
Hi, someone told me it's `Jun 19, 2023, 18:00 IST`

**Comment by shuklaritvik06** [comment]
> Hi, someone told me it's `Jun 19, 2023, 18:00 IST`

Yes it is Today 18 IST
